devdoc/userdoc/copilot/CopilotPromptTips.md

```diff
@@ -0,0 +1,84 @@
+# Copilot Prompt Tips
+Unlock the full potential of GitHub Copilot with these carefully curated prompts designed to streamline your workflow and enhance productivity. 
+
+Whether you're summarizing complex texts, brainstorming innovative ideas, or creating detailed guides, these prompts will help you get the most out of Copilot's capabilities. 
+
+Dive in and discover how these simple yet powerful prompts can save you time and effort in your daily tasks.
+
+## Generic Prompts
+
+Here are a few simple prompts that can save time:
+
+- Provide a brief summary of the text below. Show the key points in a bullet list.
+
+- List [N (number)] ways to [accomplish a specific goal or solve a problem]? Include a short description for each approach in a bullet list.
+
+  - **Example**: List 10 ways to reduce redundancy in text. Include a short description for each approach in a bullet list.
+
+- What are the key differences/similarities between [concept A] and [concept B]? Present the information in a table format.
+
+  - **Example:** What are the key differences/similarities between a quicksort ad a bubble sort? Present the information in a table format.
+
+- Explain [complex topic] in simple terms. Use analogies or examples to help make it easier to understand.
+
+  - **Example:** Explain generative AI in simple terms. Use analogies or examples to help make it easier to understand. 
+
+- Brainstorm [N (number)] ideas for [a specific project or topic]? Include a short description for each idea in a bullet list.
+
+  - **Example:** Brainstorm 7 ideas for an instructional video for new programmers? Include a short description for each idea in a bullet list.
+
+- Create a template for [a specific type of document, such as a business email, proposal, etc.]. Include the key elements to include in a bullet list.
+
+  - **Example:** Create a template for product version release announcement. Include the key elements to include in a bullet list.
+
+- Write a step-by-step guide on how to [specific task or procedure]. Number the steps to improve clarity.
+
+  - **Example:** Write a step-by-step guide on how to make a PB & J sandwich. Number the steps to improve clarity.
+
+## Specific Prompts
+
+These prompts are more focused and are meant to accomplish a specific purpose. They are included here to spark your imagination.
+
+If you think of others that could be included here to share witht he world, please send your ideas to <docfeedback@hpccsystems.com>. 
```

Comment:
spelling witht he -> with the

Reply:
Fixed the typo. Creating a Jira to enhance this article with tips to save prompt history and other enhacements